### ISSUE TITLE
Rule: label changes (part 1)

### ISSUE THREAD

**Issue by jkodu** [dequelabs/axe-core#1148]
The following test cases should yield desired result

1) Fail: `aria-label` with empty text string
```html
<div aria-label=" " role="combobox">England</div>
```

2) Fail: Label does not exist.
```html
<div aria-labelledby="non-existing" role="combobox">England</div>
```

3) Fail: Implicit label not supported on div elements.
```html
<label>
  first name
  <div role="textbox" data-rule-target></div>
</label>
```

4) Fail: Explicit label not supported on div elements.
```html
<label for="lastname">first name</label>
<div role="textbox" id="lastname" data-rule-target></div>
```

**Comment by jkodu** [comment]
The below PR's tackled the above issue
- https://github.com/dequelabs/axe-core/pull/1450
- https://github.com/dequelabs/axe-core/pull/1610
